packages/material-ui-styles/src/makeStyles/makeStyles.test.js

```diff
@@ -39,7 +39,7 @@ describe('makeStyles', () => {
   let generateClassName;
 
   before(() => {
-    mount = createMount();
+    mount = createMount({ strict: undefined });
```

Comment:
There were some very confusing fails. It's probably not strict mode ready at all.